schema.sql

```diff
@@ -12,6 +12,8 @@ create table if not exists retrieval_events(
 
 create table if not exists aggregate_retrieval_events(
   retrieval_id uuid not null UNIQUE,
+  root_cid character varying(256),
+  url_path character varying(256),
```

Comment:
Should we make this a `text` type instead? This has the potential to be fairly long since we're having to rely on user specified path string lengths.

Reply:
yeah, good point, any idea what overhead `text` is going to create tho?

Reply:
agree and there's no material difference in perf according to postgresql docs so changed.